### ISSUE TITLE
getting "E: unable to locate package libturbojpeg-dev" error while installing dependencies

### ISSUE THREAD

**Issue by h-i-r-a** [DIYBookScanner/spreads#180]
Hello,

While trying to install the dependencies using

> sudo apt-get install python2.7-dev python-pip build-essential pkg-config libffi-dev     libturbojpeg-dev     libmagickwand-dev python-cffi

I am getting the error 

> E:unable to locate package libturbojpeg-dev

I am using Ubuntu 14.04 Trusty Tahr in VirtualBox , could anyone please help me with this?

PS: On looking up packages in Ubuntu 14.04, it seems as if this package is available with a [different name](https://launchpad.net/ubuntu/trusty/+source/libjpeg-turbo), could that be why I am getting this error? I am new to Linux and everything, any help would be greatly appreciated. Thank you.

**Comment by boredland** [comment]
Install libturbojpeg:
```
sudo apt-get install libturbojpeg
```
And create a symlink (i made that easy for you):
```
turbojpeg=`sudo find / -type f -name "libturbojpeg.so.0.0.0"`
sudo ln -s $turbojpeg ${turbojpeg%.0.0.0}
```

**Comment by h-i-r-a** [comment]
@boredland Thanks very much for the reply! I used both of those instructions but it still gives exactly the same error (I am attaching a screenshot). Is there a possibility that "libturbojpeg-dev" exists under a different name ? Also, could you please explain whats the purpose of the above symlink ?

![screenshot from 2015-03-24 09 39 20](https://cloud.githubusercontent.com/assets/8281055/6798966/f990d68c-d209-11e4-96fb-d0df1d61901c.png)

**Comment by boredland** [comment]
You don't have to install libturbojpeg-dev anymore ;)

2015-03-24 10:42 GMT+01:00 h-i-r-a <notifications@github.com>:

**Comment by h-i-r-a** [comment]
Ah! But why not? Should I just ignore the above 'unable to locate' message then ?

**Comment by boredland** [comment]
just enter "sudo apt-get install python2.7-dev python-pip build-essential
pkg-config libffi-dev libmagickwand-dev python-cffi"

2015-03-24 16:33 GMT+01:00 h-i-r-a <notifications@github.com>:

**Comment by h-i-r-a** [comment]
Ah, that worked ! thank you! Is this something that can be noted down if for example one wanted to make the documentation for the spreads better for newbies?

**Comment by boredland** [comment]
Even though I totally agree: Since I still can't get spreads to work
properly myself, I think I'm in the wrong position.
Another hint for you: since you're working on regular ubuntu, you'll need
to disable the gphoto2-automounting and create a udev-rule. I can write the
commands down for you later, if you want.

2015-03-26 5:24 GMT+01:00 h-i-r-a <notifications@github.com>:

**Comment by h-i-r-a** [comment]
Yeah, that would be great if you could do that. Why would I need to disable though ?

I am getting another error now after trying to install the next set of dependencies, if you could help with that : 
![screenshot from 2015-03-26 04 25 30](https://cloud.githubusercontent.com/assets/8281055/6853943/f7ca0610-d3e5-11e4-8ca2-1c8fab03c79a.png)

Actually, I am trying to look into the DIYBookScanner project, to see how it works, and maybe to be able to contribute to the documentation or the web or post-processing aspect, or just to be able to understand it, and make it work at the moment, maybe I should start a separate post to inquire how I should go about. First have to go through the entire current documentation.

**Comment by jbaiter** [comment]
> Yeah, that would be great if you could do that. Why would I need to disable though ?

Because the gvfs service occupies the USB devices that the drivers want to talk to. With the service running, it can happen that you're unable to access the cameras.

> Actually, I am trying to look into the DIYBookScanner project, to see how it works, and maybe to be able to contribute to the documentation or the web or post-processing aspect, or just to be able to understand it, and make it work at the moment, maybe I should start a separate post to inquire how I should go about. First have to go through the entire current documentation.

If you have any questions about the architecture, reasons for why something was implemented the way it is and general pointers, just write me an email, it's on my GitHub profile page :-)

**Comment by markvdb** [comment]
I haven't checked whether it's still relevant or not, but the package name that counts when installing is the binary package name. See the list of binary packages mentioned at https://launchpad.net/ubuntu/trusty/+source/libjpeg-turbo , and https://launchpad.net/ubuntu/trusty/amd64/libjpeg-turbo8-dev. So you may want to try:
<pre>
sudo apt-get install  libjpeg-turbo8-dev
</pre>

**Comment by h-i-r-a** [comment]
> If you have any questions about the architecture, reasons for why something was implemented the way it is and general pointers, just write me an email, it's on my GitHub profile page :-)

@jbaiter Thank you, I've been wondering about how to begin to understand the architecture, and how it ties together, and how to approach it if for example one wanted to contribute towards documentation, I will drop you an email after some experimenting around ! :) 

@markvdb That worked! Thanks very much! It doesn't give the other error now either.

**Comment by boredland** [comment]
@jbaiter I think this could be closed for good.
